modules/openapi-generator/src/main/resources/JavaJaxRS/cxf-ext/pojo.mustache

```diff
@@ -46,7 +46,7 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}}{{#vendorExtensi
   @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
 {{/isDate}}
 {{#isDateTime}}
-  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'hh:mm:ss.SSSX")
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX")
```

Comment:
according to https://stackoverflow.com/a/78893569/677735, `hh` should be used instead

did you test locally to confirm the fix?